### DISCUSSION TITLE
Backup and Restore shows black images

### DISCUSSION THREAD

**Discussion by hhaim** [photoprism/photoprism#1753] [Q&A] [answered]
**First Try**
* I have two Linux servers, A (40 cores/64GB  ) and B (4/8GB) 
* Run the docker-compose daemon on NFS remote files (~60K images) on a A -- finished after 10 hours
* Stop the daemon on A and rsync add the files into to B 
* Start the daemon on B
* All the info is there *but* the images are black (if I press on a image or video I get the right image or video)
 
**Second Try**

* Backup the yaml and sql from A and restore it in B 
* Same as "first try" (black images) 

Any idea where the web small images are stored? Is it part of the mariadb? why it wasn't transferred from A->B

**Comment by graciousgrey** [comment, marked as answer]
Make sure to add the storage folder to B as well. 
Thumbnails are stored in storage/cache/thumbnails. See: https://docs.photoprism.org/user-guide/advanced/folders/

**Note:** Your storage path must NOT be located in your originals path. Otherwise your thumbnails will be indexed as well!

**Comment by hhaim** [comment]
@graciousgrey thanks. 
I missed this in docketfile,. It means that the backup is not full. It would be nice to create a tar that replicate A to B.
